src/abs.ott

```diff
@@ -64,7 +66,7 @@ e :: e_ ::=
   | fn ( e1 , ... , en ) :: :: fn_call
     {{ com function call }}
   | e [ x1 |-> y1 , ... , xn |-> yn ] :: M :: subst_var
-    {{ coq (my_subst [[e]] [[x1 |-> y1 ... xn |-> yn]]) }} % FIXME
+    {{ coq (my_subst [[e]] [[x1 |-> y1 ... xn |-> yn]]) }}
```

Comment:
```suggestion
    {{ coq (e_var_subst [[e]] [[x1 |-> y1 ... xn |-> yn]]) }}
```
Maybe some more descriptive name is better now that that it's actually defined properly?